### ISSUE TITLE
Centos 7 dependancies documentation incomplete

### ISSUE THREAD

**Issue by palanjam** [HandBrake/HandBrake-docs#101]
In the file: source/docs/en/latest/developer/install-dependencies-centos.markdown

The build instructions don't work for Centos 7. I've tried on a vanilla 7.7.1908 minimal install. There seem to be multiple issues that arise. When I run ./configure for HandBrake, I first get a couple dependencies with versions below the minimum. Two of them are automake < 1.15 and pkg-config < 0.29. I tried compiling both of them from source but then it breaks the HandBrake ./configure script since it is trying to use the system default pkg-config. I've tried setting the PKG_CONFIG_PATH env variable but that seems to get overwritten in Handbrake's configure. 

I've spent hours getting it to work and that's where I've ended up. The pkg-config can't locate the opus libraries because it is wiping the PKG_CONFIG_PATH for its own and can't find opus in /usr/lib64/pkgconfig

**Comment by bradleysepos** [comment]
I have two CentOS VMs for testing, 7 minimal/console and 7 desktop/GNOME. I'm going to set up a fresh minimal/console to check and resolve any issues. Thanks for your patience.

**Comment by bradleysepos** [comment]
It looks like some dependencies have changed since I last tested CentOS 7, notably `lame-devel` is now provided by EPEL, `cmake` was not listed (either our typo or it's no longer part of the Development Tools group`), and `bzip2-devel` was not pulled in by other packages. I've updated the documentation accordingly, and also fixed a minor typo (`..` -> `cd ..`), in 5ae244728cdaf55c7258a87ce11b75dad197150c.

Additionally, I relaxed the `automake` and `pkg-config` version requirements in https://github.com/HandBrake/HandBrake/commit/eed5cbdd841ea1a86ad104eb02ee38d8e462dd07 (and https://github.com/HandBrake/HandBrake/commit/1772b242f7b6a3856b2d58bbec0c74abaa77a225 on 1.3.x branch).

For clarity, the full command set that worked for me, and is now reflected in the documentation, is as follows:

```
sudo yum update  # reboot for kernel if desired
sudo yum groupinstall "Development Tools"
sudo yum install bzip2-devel cmake fribidi-devel git jansson-devel libogg-devel libsamplerate-devel libtheora-devel libvorbis-devel libxml2-devel numactl-devel python3 speex-devel xz-devel
sudo yum install epel-release
sudo yum install lame-devel libass-devel meson ninja-build
sudo yum localinstall $(curl -L -s 'https://dl.fedoraproject.org/pub/epel/6/x86_64/Packages/o/' | grep -Eo 'opus-[^">]+\.x86_64\.rpm' | sort -u | awk '{ print "https://dl.fedoraproject.org/pub/epel/6/x86_64/Packages/o/"$0 }')
sudo curl -L 'https://nasm.us/nasm.repo' -o /etc/yum.repos.d/nasm.repo
sudo yum install nasm
sudo yum remove libvpx-devel
curl -L https://github.com/webmproject/libvpx/archive/v1.8.1.tar.gz -o libvpx-1.8.1.tar.gz
tar -xf libvpx-1.8.1.tar.gz
cd libvpx-1.8.1
./configure --enable-shared --enable-static --as=nasm --enable-pic --enable-runtime-cpu-detect --enable-vp8 --enable-vp9
make -j$(nproc)
sudo make install
cd ..
export CFLAGS="${CFLAGS:-} -I/usr/local/include"
export LDFLAGS="${LDFLAGS:-} -L/usr/local/lib"
echo 'export CFLAGS="${CFLAGS:-} -I/usr/local/include"' >> "${HOME}/.bashrc"
echo 'export LDFLAGS="${LDFLAGS:-} -L/usr/local/lib"' >> "${HOME}/.bashrc"
if ! grep '\/usr\/local\/lib' /etc/ld.so.conf >/dev/null 2>&1; then echo '/usr/local/lib' | sudo tee --append /etc/ld.so.conf; sudo ldconfig; fi
sudo yum localinstall --nogpgcheck https://download1.rpmfusion.org/free/el/rpmfusion-free-release-7.noarch.rpm
sudo yum install x264-devel
git clone https://github.com/HandBrake/HandBrake.git && cd HandBrake
./configure --disable-gtk --launch-jobs=$(nproc) --launch
```

Hopefully I've pasted that all correctly. Apologies for the confusion and do let us know if anything else seems amiss.

**Comment by bradleysepos** [comment]
Aside, while the `PKG_CONFIG_PATH` issue shouldn't be an issue after the fixes here, I do have a pull request open to improve this behavior in the future (https://github.com/HandBrake/HandBrake/pull/2712). Since it's a change in behavior, the PR targets HandBrake 1.4.0, whereas the fixes here for CentOS 7 are in for HandBrake 1.3.2.

**Comment by palanjam** [comment]
Thank you. That worked for me as well.
